modules/integration-tests/src/main/java/org/apache/fluo/integration/impl/TransactorIT.java

```diff
@@ -111,9 +107,7 @@ public void testFailures() throws Exception {
     assertNumOpen(0);
     Assert.assertFalse(checkExists(t1));
 
-    // Test for exception to be called
-    exception.expect(IllegalStateException.class);
-    t1.getTransactorID().getLongID();
+    Assert.assertThrows(IllegalStateException.class, () -> t1.getTransactorID().getLongID());
```

Comment:
This looks fine... it's equivalent. But, one improvement that could be made is: if the expected exception is coming from `.getLongID()`, then the object returned by `t1.getTransactorID()` could be declared on the line prior to the `assertThrows`. If the expected exception is coming from `t1.getTransactorID()`, then `.getLongID()` can be dropped from the assert statement. That just makes it more clear where the exception is expected.

Reply:
Good idea. Addressed in c769c97